examples/sparql/sparql.pro

```diff
@@ -1,13 +1,16 @@
 TEMPLATE      = subdirs
 
 SUBDIRS       = simple \
-                dbpedia \
-                qmlquerymodel \
                 qmlbindings \
-                querymodel \
                 iteration \
                 asynctracker \
-                contacts
+
+# QWidget based examples disabled for now:
```

Comment:
You could put these inside a qtHaveModule(widgets) { } type condition which would prevent them being built in OBS where the module wouldn't be installed without it being added to dependencies. But then they would be built in an SDK that happened to have them installed and then you'd get errors because they're not packaged.

I don't know if there's value in packaging examples at all. The entire examples directory could easily be excluded.

Reply:
To be honest I'm not sure do any of the examples even work properly here. Some have been looking for contacts which weren't there even before(?), some are using the plain tracker plugin (not tracker_direct) which doesn't work anymore. Some even doing some hard-coded tracker1 d-bus.

Could be an option just to delete everything here, dunno.